vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -34,15 +34,18 @@
     <v-expand-transition>
       <v-form v-if="editing" ref="addressForm" name="address-form" lazy-validation>
         <div class="form__row">
-          <!-- NB: "name" attribute is needed for moveElementId() -->
-          <v-text-field autocomplete="address-complete"
+          <!-- NB1: AddressComplete needs to be enabled each time user clicks in this search field.
+               NB2: Only process first keypress -- assumes if user moves between instances of this
+                   component then they are using the mouse (and thus, clicking). -->
+          <v-text-field autocomplete="off"
                         filled
                         class="street-address"
-                        name="street-address"
+                        :id="streetAddressId"
```

Comment:
Using a unique ID here (and for country) does away with the old "search for name and move the id" code that didn't support multiple instances of this component anyway. And it's simpler!